app/middlewares/checkAuthentication.js

```diff
@@ -20,7 +20,7 @@ const decode = token => {
         }
 
         return models.session.findOne({ where: { userId: user.id } }).then(session => {
-          if (!session || session.id !== result.token) {
+          if (!session || session.id !== result.token || result.expireTime < new Date()) {
```

Comment:
Since we have time, why don't you try [moment.js](https://momentjs.com/).
Handling dates in js is a bitch, and moment helps a lot with that.